### ISSUE TITLE
Issue upgrading to 0.9.1

### ISSUE THREAD

**Issue by Sanoken** [quisquous/cactbot#348]
Hi - whenever I try to upgrade the CactbotOverlay.dll to 0.9.1, I get the error below. If I roll the dll back to 0.8.7, it works fine.  I also copied over all the new files in the cactbot folder, and the new timelines work fine with the older dll. Am I missing another dependency?

2/4/2019 9:08:26 PM: Error: LoadAddons: D:\FFXIV Stuff\Advanced Combat Tracker\OverlayPlugin\addons\CactbotOverlay.dll: System.IO.FileLoadException: Could not load file or assembly 'file:///D:\FFXIV Stuff\Advanced Combat Tracker\OverlayPlugin\addons\CactbotOverlay.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
File name: 'file:///D:\FFXIV Stuff\Advanced Combat Tracker\OverlayPlugin\addons\CactbotOverlay.dll' ---> System.NotSupportedException: An attempt was made to load an assembly from a network location which would have caused the assembly to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. See http://go.microsoft.com/fwlink/?LinkId=155569 for more information.
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadFrom(String assemblyFile, Evidence securityEvidence, Byte[] hashValue, AssemblyHashAlgorithm hashAlgorithm, Boolean forIntrospection, Boolean suppressSecurityChecks, StackCrawlMark& stackMark)
   at System.Reflection.Assembly.LoadFrom(String assemblyFile)
   at RainbowMage.OverlayPlugin.PluginMain.LoadAddons()

**Comment by quisquous** [comment]
Agreed.  Unblocking is very likely to be the issue here.

**Comment by DamienRamirez** [comment]
I actually had a issue with this build as well, I did unblock it before unzipping but in Windows 1809....best way i can explain it, Reblocked it when i moved it into c:/Programme files, i had to manually unblock the .dll & it worked. I cheaked the original .zip in my downdloads folder & it was unblocked. & the .dll was also unblocked. Just a FYI i did a rince repeat download install to check if it would happen again & it didn't. Seems like a 1 off Windows things.
